app/src/main/java/uk/co/ribot/androidboilerplate/util/RxUtil.java

```diff
@@ -0,0 +1,12 @@
+package uk.co.ribot.androidboilerplate.util;
+
+import rx.Subscription;
+
+public class RxUtil {
+
+    public static void unsubscribe(Subscription subscription) {
+        if((subscription != null) && (!subscription.isUnsubscribed())) {
```

Comment:
There are a few unnecessary brackets I believe.
Also there should be a space between the if and the first bracket:
```java
if (subscription != null && !subscription.isUnsubscribed()) {
```